src/Common/Uuid/UuidRegistry.php

```diff
@@ -47,6 +47,7 @@ class UuidRegistry
         'form_clinical_notes' => ['table_name' => 'form_clinical_notes'],
         'form_encounter' => ['table_name' => 'form_encounter'],
         'form_vitals' => ['table_name' => 'form_vitals'],
+        'form_observation' => ['table_name' => 'form_observation'],
```

Comment:
@sjpadgett , I forgot to let you know about making this change for table(s) that you have included uuid columns on.